src/app/serviceForm/openingHours/ServiceOpeningHours.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+
+import Header from '../../../components/header';
+import Button from '../../../components/button';
+import Selector from '../../../components/selector';
+import Input from '../../../components/input';
+
+const inputPlaceholder =
+  'e.g. Drop-in opens at 6pm, but you should be there by 4pm if you want to get in, etc';
+class ServiceOpeningHours extends Component {
+  state = { active: -1, weekdays: {} };
+
+  onSelect = active => this.setState({ active });
+
+  onWeekday = (index) => {
+    const { weekdays } = this.state;
+    const value = weekdays[index];
+    this.setState({ weekdays: { ...weekdays, [index]: !value } });
+  };
+
+  render() {
+    const { active, weekdays } = this.state;
+    return (
+      <div className="w-100">
+        <Header className="mb-3">When is this service available?</Header>
+        <Selector fluid>
+          <Selector.Option active={active === 0} onClick={() => this.onSelect(0)}>
+            This service is 24/7
+          </Selector.Option>
+          <Selector.Option active={active === 1} onClick={() => this.onSelect(1)}>
+            This service is <strong>not</strong> 24/7
+          </Selector.Option>
+        </Selector>
+        {active === 0 && (
+          <div>
+            <p>Is there anything else you would like to add about their opening times?</p>
+            <Input fluid placeholder={inputPlaceholder} />
+          </div>
+        )}
+        {active === 1 && (
+          <div>
+            <p>Select the days and times this service is available</p>
+            <Selector fluid>
+              <Selector.Option active={weekdays[0]} onClick={() => this.onWeekday(0)}>
```

Comment:
It would probably make sense for these weekday numbers to match the ones the API and DB expect (and generally common standards for this), where Monday is 1, and Sunday is 7.